boards/px4/fmu-v6x/default.px4board

```diff
@@ -6,7 +6,6 @@ CONFIG_BOARD_SERIAL_GPS2="/dev/ttyS7"
 CONFIG_BOARD_SERIAL_TEL1="/dev/ttyS6"
 CONFIG_BOARD_SERIAL_TEL2="/dev/ttyS4"
 CONFIG_BOARD_SERIAL_TEL3="/dev/ttyS1"
-CONFIG_BOARD_SERIAL_EXT2="/dev/ttyS3"
```

Comment:
@baumanta CONFIG_BOARD_SERIAL_EXT2 keeps getting stripped out of v6x, I'm not sure what's missing.